packages/app/src/pages/stakers/[slug].tsx

```diff
@@ -219,6 +219,8 @@ const TokenWrapper: FC<{ config?: TenderizeConfig }> = (props) => {
   const dappConfig: Config = {
     pollingInterval: 2500,
     readOnlyUrls: props.config?.chainUrlMapping,
+    // readOnlyChainId: ChainId.Hardhat,
+    // multicallAddresses: { 31337: "0x21dF544947ba3E8b3c32561399E88B52Dc8b2823" },
```

Comment:
let's either remove or uncomment depending on what is needed

If this is for local testing let's add a comment that this is for local testing otherwise

We could also simply add it to the generic config by adding a new config object `MULTICALL_ADDRESSES` as well...